docs/guides/migration-of-compute.md

```diff
@@ -0,0 +1,217 @@
+│ ID              │ cbdb7256-30d2-8dc4-288c-d7f58bc9e308 │
+│ Name            │ privnet                              │
+│ Description     │ Private Network                      │
+│ Zone            │ ch-gva-2                             │
+│ Type            │ manual                               │
+┼─────────────────┼──────────────────────────────────────┼
+```
+
+In our case, it's `cbdb7256-30d2-8dc4-288c-d7f58bc9e308`. Let's import this resource:
+
+```bash
+$ terraform import exoscale_private_network.my_network cbdb7256-30d2-8dc4-288c-d7f58bc9e308@ch-gva-2
+exoscale_private_network.my_network: Importing from ID "cbdb7256-30d2-8dc4-288c-d7f58bc9e308@ch-gva-2"...
+exoscale_private_network.my_network: Import prepared!
+  Prepared exoscale_private_network for import
+exoscale_private_network.my_network: Refreshing state... [id=cbdb7256-30d2-8dc4-288c-d7f58bc9e308]
+
+Import successful!
+
+The resources that were imported are shown above. These resources are now in
+your Terraform state and will henceforth be managed by Terraform.
+```
+
+As you can see, and as expected, the import process imported `exoscale_private_network.my_network`.
+We also import  `exoscale_compute_instance` the same way. To find instance ID, we must use CLI again:
+
+```bash
+$ exo c i show my-instance
+┼──────────────────────┼──────────────────────────────────────┼
+│   COMPUTE INSTANCE   │                                      │
+┼──────────────────────┼──────────────────────────────────────┼
+│ ID                   │ 0e28952c-1ae9-45fc-9f6d-0a9710175fd8 │
+│ Name                 │ my-instance                          │
+│ Creation Date        │ 2023-07-21 16:29:28 +0000 UTC        │
+│ Instance Type        │ standard.tiny                        │
+│ Template             │ Linux Ubuntu 22.04 LTS 64-bit        │
+│ Zone                 │ ch-gva-2                             │
+│ Anti-Affinity Groups │ n/a                                  │
+│ Deploy Target        │ -                                    │
+│ Security Groups      │ default                              │
+│ Private Instance     │ No                                   │
+│ Private Networks     │ privnet                              │
+│ Elastic IPs          │ n/a                                  │
+│ IP Address           │ 194.182.161.116                      │
+│ IPv6 Address         │ -                                    │
+│ SSH Key              │ -                                    │
+│ Disk Size            │ 10 GiB                               │
+│ State                │ running                              │
+│ Labels               │ n/a                                  │
+│ Reverse DNS          │                                      │
+┼──────────────────────┼──────────────────────────────────────┼
+```
+
+Instance ID is `0e28952c-1ae9-45fc-9f6d-0a9710175fd8` and we can import it:
+
+```bash
+$ terraform import exoscale_compute_instance.my_instance 0e28952c-1ae9-45fc-9f6d-0a9710175fd8@ch-gva-2
+
```

Comment:
What about taking advantage of the new features from 1.5?

https://developer.hashicorp.com/terraform/language/import

Reply:
Thanks for suggestion @greut . I like the import blocks and imo it should be a recommended way to perform imports in the future. But I'd stick with classic import for this particular guide to be inline with the security group migration guide and give a bit more time to everyone to upgrade to 1.5+ version of terraform.